### PR TITLE
tp-qemu: Add different logical/physical_block_size for a boot case in…

### PR DESCRIPTION
**Pull request by qunfang**
Add different logical/physical_block_size parameters for a boot case in slof feature (ppc only).
ID: 1324355

Signed-off-by: qzhang@redhat.com

### CODE REVIEW

File: qemu/tests/cfg/slof_boot.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- slof_boot: install setup image_copy unattended_install.cdrom
+    type = slof_boot
```
Comment (spcui) [inline qemu/tests/cfg/slof_boot.cfg]: @qunfang not found slof_boot.py, typo here?

**Comment by qunfang** [comment]
@spcui yeah, it's a typo, I have updated the cfg file and re-test pass, please help review it. Thanks.

File: qemu/tests/cfg/slof_boot.cfg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- slof_boot: install setup image_copy unattended_install.cdrom
+    type = boot
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
```
Comment (xutian) [inline qemu/tests/cfg/slof_boot.cfg]: Do we need to increase login_timeout if both pysical_block_size and logical_block_size are 512?

File: qemu/tests/cfg/slof_boot.cfg [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    variants:
+        - with_sec_size_4096_512:
+            physical_block_size=4096
```
Comment (spcui) [inline qemu/tests/cfg/slof_boot.cfg]: OK, the code likes good to me, and passed by testing with correct command line,
but pls keep the coding style, the line miss some space char, "physical_block_size = 4096" is better.

File: qemu/tests/cfg/slof_boot.cfg (hunk 2 above)
Comment (qunfang) [inline qemu/tests/cfg/slof_boot.cfg]: In my all experiments, the overall two cases could finish within short time (1~2 min). So is there other concern to increase the login_timeout?

Comment (xutian) [inline qemu/tests/cfg/slof_boot.cfg]: is it a ppc only test case?

File: qemu/tests/cfg/slof_boot.cfg (hunk 3 above)
Comment (qunfang) [inline qemu/tests/cfg/slof_boot.cfg]: Thanks for the comment, I have re-updated it.

File: qemu/tests/cfg/slof_boot.cfg (hunk 2 above)
Comment (qunfang) [inline qemu/tests/cfg/slof_boot.cfg]: Yes, it is.

Comment (xutian) [inline qemu/tests/cfg/slof_boot.cfg]: so only ppc64 ppc64le are required

Comment (spcui) [inline qemu/tests/cfg/slof_boot.cfg]: need add "only ppc64 ppc64le" in config.

**Comment by CongSmile** [comment]
Hi @qunfang 

There is a similar case called "check_block_size".
I think this case can cover your requests but may needs some updates.

Could you help check it?
Thanks.

File: qemu/tests/cfg/slof_boot.cfg (hunk 2 above)
Comment (qunfang) [inline qemu/tests/cfg/slof_boot.cfg]: Updated and re-test pass. :)

**Comment by spcui** [comment]
Acked-by: scui@redhat.com

**Comment by CongSmile** [comment]
The request of "slof_boot" is different from case "check_block_size".

LGTM,
Acked-by: Cong Li <coli@redhat.com>

**Comment by PandaWei** [comment]
LGTM,  
But It's better to attach the result for new case.

**Comment by CongLi** [comment]
Merged.

Thanks @qunfang @PandaWei @spcui @xutian

**Pull request merged**